JustSaying.Sample.Restaurant.KitchenConsole/Program.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Threading.Tasks;
+using JustSaying.Sample.Restaurant.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace JustSaying.Sample.Restaurant.KitchenConsole
+{
+    internal class Program
+    {
+        public static async Task Main()
+        {
+            Console.Title = "KitchenConsole";
+
+            await new HostBuilder()
+                .ConfigureAppConfiguration((hostContext, config) =>
+                {
+                    config.SetBasePath(AppContext.BaseDirectory);
```

Comment:
Is there a method on `HostBuilder` anywhere that does all this by default? `WebHostBuilder` does similar (or comes preconfigured that way), so would shave off a few lines of code if it's not needed or there's a one-liner.

Reply:
Just removing it seems to work...
executing `dotnet run` from the projects folder works as does ` dotnet run --project samples/JustSaying.Sample.Restaurant.KitchenConsole/JustSaying.Sample.Restaurant.KitchenConsole.
csproj`